src/main/java/net/dv8tion/jda/webhook/WebhookClientBuilder.java

```diff
@@ -62,6 +65,29 @@ public WebhookClientBuilder(final long id, final String token)
     }
 
     /**
+     * Creates a new WebhookClientBuilder with the provided webhook URL
+     *
+     * @param  url
+     *         The URL of the webhook. May be directly copied from Discord's UI
+     *         <br>Example: {@code https://discordapp.com/api/webhooks/123456789012345678/my-webhook-token}
+     *         <br>This constructor also parses URLs pointing to subdomains of {@code discordapp.com}
+     *
+     * @throws java.lang.IllegalArgumentException
+     *         If the provided URL is {@code null}
+     *         or is incorrectly formatted
+     */
+    public WebhookClientBuilder(@Nonnull String url)
+    {
+        Matcher matcher = WEBHOOK_PATTERN.matcher(url);
+        if(!matcher.find()) {
+            throw new IllegalArgumentException("Failed to parse webhook URL");
+        }
+
+        this.id = Long.parseLong(matcher.group(1));
+        this.token = matcher.group(2);
```

Comment:
Use the existing constructor with the results, that should be `MiscUtil.parseSnowflake(...)`

Reply:
There's no clean way of using the same constructor
![image](https://user-images.githubusercontent.com/2582617/31850809-808a8bde-b659-11e7-923e-0ed6f49b3730.png)

Reply:
Understandable, still use the proper parse method.